package.json

```diff
@@ -44,7 +44,7 @@
     "start-worker": "node worker/index.js",
     "test": "npm run reset-coverage && npm run test-all && npm run report",
     "test-all": "npm run test-main && npm run test-extra && npm run test-collector",
-    "test-api": "npm run test-with-coverage -- --recursive tests/api",
+    "test-api": "npm run test-with-coverage -- --recursive tests/api/",
```

Comment:
why is this line changing?
did you intend to update package-lock.json in this PR as well?

Reply:
reverted this line as well as package-lock.json